apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.html

```diff
@@ -1,4 +1,4 @@
-<div [ngClass]="{'custom-input-field communication-input': !appearance }">
+<div [ngClass]="{'custom-input-field': !appearance }">
```

Comment:
please check for campaign and fixed this issue

Reply:
Done